WordPress/Classes/Networking/PeopleRemote.swift

```diff
@@ -43,7 +43,8 @@ class PeopleRemote: ServiceRemoteWordPressComREST {
 
         wordPressComRestApi.GET(path, parameters: parameters, success: { (responseObject, httpResponse) in
             guard let response = responseObject as? [String: AnyObject],
-                      people = try? self.peopleFromResponse(response, siteID: siteID, type: User.self) else
+                users = response["users"] as? [[String: AnyObject]],
+                people = try? self.peopleFromResponse(users, siteID: siteID, type: User.self) else
```

Comment:
Careful here.  I haven't tested this but there's a pretty good chance you may have a retain cycle here since:

`self` -(strong)-> `wordPressComRestApi` -(strong)-> `self`

I suggest `weak self` inside these blocks just to stay safe.  If `self` is nil, you can drop the response.